R/Instance.R

```diff
@@ -1,166 +1,108 @@
-create_instance_class <- function(instance_settings) {
-  super <- NULL # satisfy R CMD check and lintr
+create_instance <- function(instance_settings) {
+  super <- NULL # satisfy linter
```

Comment:
indeed... when an R6Class is created directly in the package, it takes into account that it shouldn't generate any warnings for `self`, `super`, `private` and `public`. However, since this R6Class is created inside a function, this condition doesn't get triggered and warnings are produced.